conn_oob.go

```diff
@@ -113,9 +113,17 @@ func newConn(c OOBCapablePacketConn) (*oobConn, error) {
 			return nil, errors.New("activating packet info failed for both IPv4 and IPv6")
 		}
 	}
+
+	var bc batchConn
+	if ibc, ok := c.(batchConn); ok {
```

Comment:
Can you add a comment here?

Reply:
Explaining why this would be desireable?

Reply:
Yeah, that applications may pass in a connection that implements `batchConn`. Just to make clear that this is a code path that's only taken if the application takes special care.

Reply:
Done, I think.
My main motive is that it would circumvent my ability to manage what is read, as ipv4.NetPacketConn ends up upwrapping the raw file descriptor and taking away any ability for me to control what is being read.

Reply:
SGTM.